modules/apps/portal-search/portal-search-ranking-web/src/main/resources/META-INF/resources/js/components/ResultsRankingForm.es.js

```diff
@@ -614,6 +620,9 @@ class ResultsRankingForm extends Component {
 	_handleUpdateAddResultIds = addedResultsDataList => {
 		const mappedData = resultsDataToMap(addedResultsDataList);
 
+		// Make sure that all added results that don't overlap with the original
+		// search results get marked as pinned and addedResult.
```

Comment:
Instead of "overlap" maybe something along the lines of "included with the original search results" or "part of"? "Overlap" seems to be like we're describing something visual.